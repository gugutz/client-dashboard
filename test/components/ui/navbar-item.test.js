import React from 'react'
import renderer from 'react-test-renderer'
import {BrowserRouter as Router} from 'react-router-dom'
import NavbarItem from '../../../src/components/ui/navbar-item'

describe('<NavbarItem /> component', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(
        <Router>
          <NavbarItem href="/home" text="Home"/>
        </Router>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
