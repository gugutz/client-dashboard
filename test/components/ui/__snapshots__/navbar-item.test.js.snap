// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavbarItem /> component should render correctly 1`] = `
<li
  className="navbar-item"
>
  <a
    href="/home"
    onClick={[Function]}
  >
    Home
  </a>
</li>
`;
